e2e/test_teams.py

```diff
@@ -101,6 +103,7 @@ def test_api_v0_update_team(team):
     re = requests.put(api_v0('teams/'+team_name), json={'name': new_team_name,
                                                         'email': email,
                                                         'slack_channel': slack,
+                                                        'slack_channel_notificationse': slack_notifications,
```

Comment:
Should this say `slack_channel_notifications` instead of `slack_channel_notificationse`?

Reply:
thankyou